SPECS/python-sphinx/python-sphinx.spec

```diff
@@ -237,21 +189,41 @@ mkdir %{buildroot}%{python3_sitelib}/sphinxcontrib
   >> sphinx.lang
 
 %check
-pip3 install more-itertools
-%pytest
+pip install --upgrade \
+  exceptiongroup \
+  filelock \
+  html5lib \
+  iniconfig \
+  tomli \
+  "pygments>=2.14"
+
+# ignoring some tests due to incompatible dependencies
+# html5lib: test_build_html, test_build_latex
+# graphviz: test_ext_graphviz, test_ext_inheritance_diagram
+%pytest \
+  --ignore tests/test_build_html.py \
+  --ignore tests/test_build_latex.py \
+  --ignore tests/test_ext_graphviz.py \
```

Comment:
Is this incompatible with our graphviz completely? Maybe sync with @SeanDougherty. (not critical, just an fyi)

Reply:
I haven't tested with the new graphviz yet. But these tests don't seem to work with the version on 3.0-dev (2.x)